torch/_torch_docs.py

```diff
@@ -2381,7 +2381,9 @@ def merge_dicts(*dicts):
 .. warning::
     Integer division using div is no longer supported, and in a future release
     div will perform true division as in Python 3. Use :func:`torch.true_divide`
-    or :func:`torch.floor_divide` (// in Python), instead.
+    or :func:`torch.floor_divide` (// in Python), instead. To mimic the historic
```

Comment:
should it be `//` instead of //?

Reply:
nit: to mimic historical integer division (C-style), use...

Reply:
SG.